src/tools/rust-analyzer/crates/profile/src/hprof.rs

```diff
@@ -83,6 +83,7 @@ pub fn heartbeat() {
     }
 }
 
+#[cfg_attr(not(bootstrap), allow(unnameable_types))]
```

Comment:
r-a is not affected by bootstrapping, the lints shouldn't trigger on rust-analyzer in the first place if its meant to be enabled for rustc